src/megachatapi_impl.cpp

```diff
@@ -964,6 +971,13 @@ void MegaChatApiImpl::sendPendingRequests()
         }
         case MegaChatRequest::TYPE_HANG_CHAT_CALL:
         {
+            if (!mClient->rtc)
+            {
+                API_LOG_ERROR("WebRTC is not initialized");
+                errorCode = MegaChatError::ERROR_ACCESS;
+                break;
+            }
+
```

Comment:
Fix `hangupAll()` method, which destroys the call instead of calling `hangup()` for each call